src/Illuminate/Events/CallQueuedListener.php

```diff
@@ -11,6 +11,13 @@ class CallQueuedListener implements ShouldQueue
 {
     use InteractsWithQueue;
 
+    /**
+     * The listener class instance
```

Comment:
This description is incorrect.

Reply:
The failed method is called because handle() throws Exception, so the $this->handler is not read in the failed method.

Reply:
@GrahamCampbell 
How should I fix this?